cat/ptx-v6.0.cat

```diff
@@ -30,7 +30,7 @@ let sr-m = ([strong-m]; (sr & loc); [strong-m])
 let po-m = ([M]; po-loc; [M])
 let morally-strong = sr-fence | po-fence | sr-m | po-m
 let fence-sc = [F & SC] ; sr; [F & SC]
-let sync-barrier = [BAR_BEFORE]; (po | (sync_bar & scta)); [BAR_AFTER]
+let sync-barrier = [F & BAR]; (sync_bar & scta); [F & BAR]
```

Comment:
- Yes the BAR seems not useful here. I'll remove both `BAR` and `F`.
- For the `scta` I think we'd better to keep it outside of `sync_bar` in case we need this relation in other models (like in vulkan the scope could have different names).

Reply:
But AFAIK in Vulkan you don't have fences with IDs, so you could still not reuse this. However I agree keeping these two separated gives us flexibility. However, you can keep them separated but not expose this to the user. See WMM.java. there you can match the name with an intersection relation.

Reply:
Accepted